frameworks/hdfs/universe/resource.json

```diff
@@ -1,7 +1,7 @@
 {
   "assets": {
     "uris": {
-      "jre-tar-gz": "https://downloads.mesosphere.com/kafka/assets/jre-8u91-linux-x64.tar.gz",
+      "jre-tar-gz": "https://s3-us-west-2.amazonaws.com/infinity-artifacts/jre-8u91-linux-x64.tar.gz",
```

Comment:
We should use a production bucket, not infinity-artifacts.

Also, let's use the user-friendly S3 URLs I referenced above.